client/state/activity-log/actions.js

```diff
@@ -163,8 +163,9 @@ export function updateRewindRestoreProgress( siteId, timestamp, restoreId, progr
 	return {
 		type: REWIND_RESTORE_UPDATE_PROGRESS,
 		...progress,
-		siteId,
+		freshness: Date.now(),
```

Comment:
I wasn't sure about the best place to do it and I think it is here.

Reducers are supposed to be pure, and introducing this in a reducer would break purity.